pipeline.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#Pipeline/integration tests for efm-mocks to setup the mocks and integrasjonspunkt to send messages to the mock. 
+
+
+#start mock services
+docker-compose up -d --build --force-recreate
+
+
+#start sending integrasjonspunkt
+docker pull difimove2/integrasjonspunkt:development
+docker run -d --name sending_integrasjonspunkt --net movemocks_default -p 9093:9093 -e SPRING_PROFILES_ACTIVE=mock -eDIFI_MOVE_SERVICEREGISTRYENDPOINT=http://wiremock:8090 -eDIFI_MOVE_NOARKSYSTEM_ENDPOINTURL=http://sa-mock:8002/p360 -eDIFI_MOVE_DPI_ENDPOINT=http://dpimock:8080/as4 -eDIFI_MOVE_DPV_ENDPOINTURL=http://mocks:8001/dpv/ -eDIFI_MOVE_FIKS_UT_ENDPOINTURL=http://mocks:8001/dpf -eDIFI_MOVE_FIKS_INN_BASEURL=http://mocks:8001/svarinn -eDIFI_MOVE_NEXTMOVE_SERVICEBUS_BASEURL=mocks:8001/dpe -eDIFI_MOVE_DPO_STREAMINGSERVICEURL=http://mocks:8001/dpo/ServiceEngineExternal/BrokerServiceExternalBasicStreamed.svc?wsdl -eDIFI_MOVE_DPO_BROKERSERVICEURL=http://mocks:8001/dpo/ServiceEngineExternal/BrokerServiceExternalBasic.svc?wsdl -eAPP_LOGGER_ENABLESSL=false difimove2/integrasjonspunkt:development
```

Comment:
APP_LOGGER_ENABLESSL er vel utfasa for ei stund sidan?

Reply:
Kanskje vi skulle omskrevet dette til en docker-compose så blir kommandoen en del lettere å lese/endre?

Reply:
Godt poeng, det skal vekk. fjernar.

Reply:
Ser det også kan legges i en eneg env-file: https://docs.docker.com/engine/reference/commandline/run/#set-environment-variables--e---env---env-file

Kanskje dette er beste løsningen?

Reply:
@larsbard  eg lagar eiga sak på å sjå på rydding.